extensions/amp-script/0.1/amp-script.js

```diff
@@ -234,7 +234,7 @@ export class AmpScript extends AMP.BaseElement {
     let container;
     if (this.element.sizerElement) {
       container = this.win.document.createElement('div');
-      this.applyFillContent(container, true);
+      applyFillContent(container, true);
```

Comment:
I know this is how it was, adding this comment to make the addition trivial
```suggestion
      applyFillContent(container, /* replacedContent */ true);
```